keyboards/handwired/jscotto/scotto40/info.json

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
Please remove all the inline comments (`// Row 1`, etc.) from this file.

Reply:
I think for sake of clarity with this being a handwired board, the comments are useful. When wiring it up, it's nice to be able to quickly see where the rows are and what matrix coordinates are needed.